.travis.yml

```diff
@@ -15,30 +15,19 @@ matrix:
     fast_finish: true
 
     include:
-    - php: 7.1
-      env:
-      - COMPOSER_FLAGS="--prefer-stable --prefer-lowest"
-
     - php: 7.2
       env:
       - COMPOSER_FLAGS="--prefer-stable --prefer-lowest"
 
-    - php: '7.1'
```

Comment:
i am fine with dropping 7.1 support, but then we need to adjust the php version in the composer.json `require` section to `^7.2` as well